crates/trade/src/lib.rs

```diff
@@ -11,11 +11,36 @@ use uuid::Uuid;
 
 pub mod cfd;
 
-/// The trade parameters defining the trade execution
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct NewOrder {
+    pub price: Decimal,
+    pub quantity: Decimal,
+    pub maker_id: String,
+    pub direction: Direction,
+}
+
+/// A match represents the matched orders of a taker and a maker and will be executed by the
+/// coordinator.
 ///
-/// Emitted by the orderbook when a match is found.
-/// Both trading parties will receive trade params and then request trade execution with said trade
-/// parameters from the coordinator.
+/// Emitted by the orderbook to the coordinator for execution when a match is found.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct MatchParams {
+    /// The taker trade information
```

Comment:
I think we need to be able to identify which order was matched because the trader might have multiple orders pending.

Reply:
Sorry, the match params ended up in the wrong commit, the trade model (introduced in the next commit) includes the order id.